plutus-ledger-api/src/PlutusLedgerApi/Common/Eval.hs

```diff
@@ -75,7 +75,7 @@ a limit to guard against scripts that run for a long time or loop.
 Note: Parameterized over the ledger-plutus-version since the builtins allowed (during decoding) differs.
 -}
 evaluateScriptRestricting
-    :: LedgerPlutusVersion
+    :: PlutusVersion
```

Comment:
I deliberately gave it the longer name. I'm trying to make it obvious that it's the _ledger_ plutus version, since this is pretty confusing already.